pkgs/by-name/sl/sly/package.nix

```diff
@@ -0,0 +1,63 @@
+{
+  lib,
+  fetchFromGitHub,
+  flutter,
+  autoPatchelfHook,
+  makeDesktopItem,
+  copyDesktopItems,
+}:
+let
+  version = "0.4.0";
+  src = fetchFromGitHub {
+    owner = "kra-mo";
+    repo = "sly";
+    rev = "v${version}";
+    hash = "sha256-P7LhhXQQDRsDQ8bZgfvWazLRMYVGhFhMTD41fgs718g=";
+  };
+in
+flutter.buildFlutterApplication {
+  pname = "sly";
+  inherit version src;
+
+  desktopItems = [
```

Comment:
Why don't we use this https://github.com/kra-mo/sly/blob/main/packaging/linux/page.kramo.Sly.desktop?